.github/workflows/test_prereleases.yml

```
@@ -57,12 +55,11 @@ jobs:
         shell: powershell
 
       - name: Install Windows OpenGL
```

```suggestion
      - name: Setup headless display
```
honestly not sure these names are needed, but...